### ISSUE TITLE
Regular expression denial-of-service (ReDoS)- Vulnerability trim-newlines

### ISSUE THREAD

**Issue by panbenson** [bugsnag/bugsnag-build-reporter-node#14]
### Describe the bug
https://nvd.nist.gov/vuln/detail/CVE-2021-33623
> The trim-newlines package before 3.0.1 and 4.x before 4.0.1 for Node.js has an issue related to regular expression denial-of-service (ReDoS) for the .end() method.

Please update to the latest version of [meow](https://github.com/sindresorhus/meow/releases/tag/v10.0.1)

### Steps to reproduce
1. Run `yarn audit`
```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ high          │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ trim-newlines                                                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=3.0.1 <4.0.0 || >=4.0.1                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ yelp-js-infra                                                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ webpack-bugsnag-plugins >                                    │
│               │ bugsnag-build-reporter > meow > trim-newlines                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://www.npmjs.com/advisories/1753                        │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

**Comment by yousif-bugsnag** [comment]
Hi @panbenson thanks for raising this. We'll look at addressing this as soon as priorities allow.

**Comment by mattdyoung** [comment]
Fixed in [v2.0.0](https://github.com/bugsnag/bugsnag-build-reporter-node/releases/tag/v2.0.0).

Note that although bugsnag-build-reported-node was not vulnerable, moving meow to v10 was necessary to resolve the security warning. Later versions of meow drop support for Node v6, so we've made the decision to drop support for Node v6 in v2.0.0.
